.github/workflows/conformance.yml

```diff
@@ -0,0 +1,80 @@
+name: Manually run upstream conformance test on Linux
+
+on:
+  workflow_dispatch:
+    inputs:
+      antrea-version:
+        description: The Antrea version to test. It could be a SHA-1 value, a branch, or a tag (e.g. a7b012b, release-1.12, v1.12.0).
+        default: main
+        required: true
+      k8s-version:
+        description: K8s version (e.g. v1.27.1). Kind's default K8s version will be used if empty.
+        required: false
+      test-suite:
+        # Check run-k8s-e2e-tests.sh for which test cases these values represent.
+        description: The test suite to run. Valid values include "whole-conformance", "conformance", "network-policy, "sig-network", and "all".
```

Comment:
There is support for an `choice` type which restricts the input to a list of pre-defined values. I believe that the user will see a drop-down menu of choices when triggering the workflow.

```yaml
type: choice
options:
    - whole-conformance
    - conformance
    - ...
```

Reply:
thanks for the tip, it's much friendlier